go/cmd/vtctldclient/command/tablets.go

```diff
@@ -557,6 +568,26 @@ func commandStopReplication(cmd *cobra.Command, args []string) error {
 	return err
 }
 
+func commandGetPermissions(cmd *cobra.Command, args []string) error {
+	alias, err := topoproto.ParseTabletAlias(cmd.Flags().Arg(0))
+	if err != nil {
+		return err
+	}
+
+	cli.FinishedParsing(cmd)
+
+	resp, err := client.GetPermissions(commandCtx, &vtctldatapb.GetPermissionsRequest{
+		TabletAlias: alias,
+	})
+	if err != nil {
+		fmt.Printf("%v\n", err)
+		return err
+	}
+	p, err := json2.MarshalIndentPB(resp.Permissions, "	")
+	fmt.Printf("%s\n", p)
```

Comment:
`cli.MarshalJSON` and also, if you get an error marshaling, you should probably not try to print